WalletWasabi/Backend/Models/FilterModel.cs

```diff
@@ -25,20 +32,19 @@ public FilterModel(SmartHeader header, GolombRiceFilter filter)
 
 	public static FilterModel FromLine(string line)
 	{
-		Guard.NotNullOrEmptyOrWhitespace(nameof(line), line);
 		string[] parts = line.Split(':');
 
 		if (parts.Length < 5)
 		{
 			throw new ArgumentException(line, nameof(line));
 		}
 
-		var blockHeight = uint.Parse(parts[0]);
-		var blockHash = uint256.Parse(parts[1]);
-		var filterData = Encoders.Hex.DecodeData(parts[2]);
-		GolombRiceFilter filter = new(filterData, 20, 1 << 20);
-		var prevBlockHash = uint256.Parse(parts[3]);
-		var blockTime = DateTimeOffset.FromUnixTimeSeconds(long.Parse(parts[4]));
+		uint blockHeight = uint.Parse(parts[0]);
+		uint256 blockHash = uint256.Parse(parts[1]);
+		byte[] filterData = Encoders.Hex.DecodeData(parts[2]);
+		Lazy<GolombRiceFilter> filter = new(() => new GolombRiceFilter(filterData, 20, 1 << 20), LazyThreadSafetyMode.ExecutionAndPublication);
```

Comment:
Yes, I was thinking the same but that is only relevant when accessing the values. So I kind of think it's an overkill, so I decided for simplicity. But if needed, it can be certainly done.